source/docs/articles/sites/domains.md

```diff
@@ -64,7 +61,7 @@ Using the provided destinations in the Site Dashboard, create the recommended DN
 To serve your site from the bare domain, you must:
 
 1. Select a DNS provider that supports CNAME flattening, such as [CloudFlare (recommended)](https://support.cloudflare.com/hc/en-us/articles/200169056-CNAME-Flattening-RFC-compliant-support-for-CNAME-at-the-root), [ClouDNS](https://www.cloudns.net/features/), or [NameCheap](https://www.namecheap.com/domains/freedns.aspx).
-2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for **both** the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
+2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for both the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
```

Comment:
@bmackinney do you mean a screenshot from a DNS provider? I think we try to avoid images from other services since they're a pain to keep up to date, but I can add this one from cloudflare if you think its needed
<img width="952" alt="screen shot 2016-02-12 at 2 47 58 pm" src="https://cloud.githubusercontent.com/assets/10119525/13022734/30b56a04-d198-11e5-8981-0b80645616bb.png">